synapse/lib/modelrev.py

```diff
@@ -790,6 +797,7 @@ async def revCoreLayers(self):
 
         layers = []
         for layr in self.core.layers.values():
+            layr: s_layer.Layer
```

Comment:
Isn't `layr` self evident enough? Seems less readable this way...

Reply:
Agree. This is just type hinting for my editor. I was planning on removing this before taking the PR out of draft :)